webpack/components/extensions/HostDetails/Tabs/__tests__/tracesTab.test.js

```diff
@@ -217,3 +217,29 @@ describe('Without tracer installed', () => {
     expect(enableTracesModalLink).toHaveClass('pf-m-in-progress');
   });
 });
+
+test('verify banner is present for old host page', async (done) => {
+  // Setup autocomplete with mockForemanAutoComplete since we aren't adding /katello
+  const autocompleteScope = mockForemanAutocomplete(nockInstance, autocompleteUrl);
+
+  const noResults = {
+    total: 0,
+    subtotal: 0,
+    page: 1,
+    per_page: 20,
+    results: [],
+  };
```

Comment:
Would prefer this as a separate fixture for consistency

Reply:
Just realized there's another test that handles this the same way. ('Can handle no traces being present') this should be updated too :)